presets/mainnet/merge.yaml

```diff
@@ -2,8 +2,8 @@
 
 # Execution
 # ---------------------------------------------------------------
-# 2**20 (= 1,048,576)
-MAX_BYTES_PER_TRANSACTION: 1048576
+# 2**24 (= 16,777,216)
```

Comment:
If we go from `2**20` to `2**30` (instead of `2**24`) we add 10 hashes work (of 64 bytes input) per transaction to mix-in zero-hashes. At 1000 transactions per block, that's 10.000 hashes. The participation registry in altair is mostly re-merkleized (sparse and many changes, condensed leaf nodes), and at 250k validators (mainnet today) (packed uint8, 32 per leaf) that translates to `(250000/32)*2-1=15624` hashes. So this compares to adding another one of those in terms of performance cost. Worth the future-compatibility I think.